tensorflow/core/grappler/optimizers/auto_mixed_precision_test.cc

```diff
@@ -93,36 +93,51 @@ void VerifyGraphsEquivalent(const GraphDef& original_graph,
 // because otherwise the optimizer will not turn clearlist nodes to float16.
 // When looking at clearlist nodes, this optimizer checks if the nodes have a
 // float16 GPU OpKernel, but without CUDA/HIP there are no GPU OpKernels at all.
-#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
+#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM || INTEL_MKL
```

Comment:
This `#if`-statement is now redundant since there's already the following at line 16

```c++
#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM || INTEL_MKL
```

I would delete it and move the comment to that `#if`-statement.

Reply:
Correct, fixed.